classes/class-sailthru-content.php

```diff
@@ -267,7 +297,7 @@ function generate_payload( $post, $post_id ) {
 		$data['author']            = get_the_author_meta( 'display_name', $post->post_author );
 		$data['date']              = $post->post_date;
 		$data['vars']['post_type'] = $post->post_type;
-		$data['spider']            = 1;
+		$data['spider']            = $spider_value=="true" ? 1 : 0;
```

Comment:
nitpick(non-blocking): Can we inline the creation of the $data array starting on line 294?

I'm not sure if PHP will be smart enough to automatically inline the creation of the $data data structure. Would be great if we could help it out a little bit. Try and use the new style array creation if you can.

Reply:
I didn't write the array creation code so I'm not that familiar with it. Not sure if I should update it either if its simply a nitpick. Same goes for the suggestion below.

Reply:
https://conventionalcomments.org/
This is the commenting scheme that Lindsay has Integrations use. It should clear up the intent behind using the labels nitpick and suggestion.

Also, the nitpick falls under the "Boy Scout Rule" from Clean Code as well. We want to leave the code better than we found it. And in the case, we get a slight performance bump. Same as below.